pkg/resource/aws/deserializer/vpc_deserializer.go

```diff
@@ -0,0 +1,43 @@
+package deserializer
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	resourceaws "github.com/cloudskiff/driftctl/pkg/resource/aws"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/zclconf/go-cty/cty"
+	"github.com/zclconf/go-cty/cty/gocty"
+)
+
+type VPCDeserializer struct {
+}
+
+func NewVPCDeserializer() *VPCDeserializer {
+	return &VPCDeserializer{}
+}
+
+func (s *VPCDeserializer) HandledType() resource.ResourceType {
+	return resourceaws.AwsVpcResourceType
+}
+
+func (s VPCDeserializer) Deserialize(recordList []cty.Value) ([]resource.Resource, error) {
```

Comment:
You kept the recordList from Route53 records, can you change it to something more inline with vpc, like `vpcList` and update all the lines below.

Reply:
Nice catch, let's update it for something more generic like `rawList` as we use `rawResource` as iterator name in the loop.